Library/Formula/terminal-notifier.rb

```diff
@@ -27,6 +27,6 @@ def install
   end
 
   test do
-    system "#{bin}/terminal-notifier", "-message", "Hello World!"
+    system "#{bin}/terminal-notifier", "-help"
```

Comment:
Hey @mikemcquaid no problem to add the version check but is there anything I can do in terminal-notifier to smoothen the process regarding testing?